Podfile

```diff
@@ -1,33 +1,42 @@
 workspace 'VimeoUpload'
-project 'Framework/VimeoUpload/VimeoUpload.xcodeproj'
-project 'Examples/VimeoUpload-iOS/VimeoUpload-iOS.xcodeproj'
-project 'Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload.xcodeproj'
+
+use_frameworks!
+platform :ios, '8.0'
 
 target 'VimeoUpload' do
-    platform :ios, '8.0'
-    use_frameworks!
-    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'feature/VIM-6917_Swift42Migration'
     project 'Framework/VimeoUpload/VimeoUpload.xcodeproj'
 end
 
-target :'VimeoUpload-iOS' do
-    platform :ios, '8.0'
-    use_frameworks!
-    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
+target 'VimeoUpload-iOS' do
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'feature/VIM-6917_Swift42Migration'
     project 'Examples/VimeoUpload-iOS/VimeoUpload-iOS.xcodeproj'
 
     target "VimeoUpload-iOSTests" do
         inherit! :search_paths
     end
 end
 
-target :'VimeoUpload-iOS-OldUpload' do
-    platform :ios, '8.0'
-    use_frameworks!
-    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
+target 'VimeoUpload-iOS-OldUpload' do
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'feature/VIM-6917_Swift42Migration'
```

Comment:
Just noting to make sure we change this before merging (once the VimeoNetworking PR is merged)